packages.yml

```
@@ -1,3 +1,4 @@
 packages:
-- package: fivetran/apple_store_source
-  version: [">=0.4.0", "<0.5.0"]
+  - git: https://github.com/fivetran/dbt_apple_store_source.git
+    revision: nov_2024_schema
+    warn-unpinned: false
```

Reminder that this will need to be the following before cutting the pre-release.
```suggestion
  - package: fivetran/apple_store
    version: 0.5.0-a1
```